docker/full-node/data/nginx/app.conf

```diff
@@ -38,32 +38,10 @@ server {
     }
 }
 
-server {
-    listen 80;
-    listen [::]:80;
-    server_name provider.ursa.earth www.provider.ursa.earth;
-
-    location /.well-known/acme-challenge/ {
-        root /var/www/certbot;
-    }
-
-    location /stub_status {
-      stub_status;
-    }
-
-
-    location / {
-      add_header content-type  application/json;
-      add_header cache-control public,max-age=31536000,immutable;
-
-      proxy_pass http://ursa:8070;
-    }
-}
-
 server {
     listen 443 ssl http2;
     listen [::]:443 ssl http2;
-    server_name ursa.earth www.ursa.earth;
+    server_name node.ursa.earth www.node.ursa.earth;
```

Comment:
What’s node.ursa.earth?

Reply:
just an example domain it'd be edited by the deployer as per readme.md